### PR TITLE
maps: Change base names of Heroes [skip ci]

### PR DESCRIPTION
**Pull request by andy5995**
Adding a user map contained in the zip file provided by @ph2708 in https://github.com/netpanzer/netpanzer/discussions/76

### CODE REVIEW

File: Heroes.opt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ObjectiveCount: 6
+Name: Looser#1
+Location: 32 15
+Name: Heores
+Location: 186 6
+Name: Looser#3
```
Comment (winrid) [inline Heroes.opt:6]: why are the base names Looser? :D

Comment (winrid) [reply Heroes.opt:6]: @ph2708 ?

**Review by winrid — APPROVED** [review]
sounds good to me

**Comment by andy5995** [comment]
Oh I see I screwed up a little. The map already existed. Anyways, we got some better names now.

**Pull request merged**